ProcessMaker/screen-builder
# FOUR-7161 Add reactivity to non defined calculated properties

## Issue & Reproduction Steps
In the attached screen the reactivity of the variable `requireInitialDepositOther` did not work until the user go back one page then return to the page 2 (where this variable is used).

**Actual behavior:** 
The variable `requireInitialDepositOther` is added by code in a calculated property `accounts`. Since this variable was not defined nor initialized it is not reactive, and since the logic of the calculated properties in v.4.3.0 was optimized (did not refresh all the screen and data all the times a calculated property is executed)

**Expected behavior:** 
It is required to support reactivity for new non defined nor initialized attributes added in a calculated property by code, this way the user can add dynamically attributes (without declaration nor initialization) that reacts immediately when the attribute is added/changed

## Solution
- Update the logic of the calculated property so it convert to reactive new attribute added (this can be done by calling the `addNonDefinedComputedAttributes` method of the screen with the new attributes).

## How to Test
- Import the screen
- Preview it
- Add a new account
- Select a product type
- Go to page 2
- Choose a transfer type
- The initial deposit field should be visible

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-7161

## Code Review Checklist
